### ISSUE TITLE
Avoid navigating through "Pods" folders

### ISSUE THREAD

**Issue by bryant1410** [bryant1410/readmesfix#12]
As noted in andreamazz/AMTagListView#63

**Comment by styfle** [comment]
Fascinating. What does the Pods folder represent? Is it created by a package manager?

**Comment by bryant1410** [comment]
Yeah, I think is created by some iOS tool, related to cocoapods, carthage and the like. I have heard it's staged on github to save time, and also to modify the "pods" if you want (the dependencies)
